### PR TITLE
[GSOC] Draw Method for Arch 

### PR DESCRIPTION
**Pull request by shishir-11**
Adds the code for the draw method for arches allowing plotting an accurate diagram for the structure.

#### Release Notes

* physics.continuum_mechanics
  * add a draw method for the Arch class

### CODE REVIEW

**Comment by shishir-11** [comment]
@Ishanned @AdvaitPote Please have a look .

File: sympy/physics/continuum_mechanics/arch.py [hunk 1]
```diff
@@ -639,3 +644,347 @@ def solve(self):
...
+        ========
+
+        >>> from sympy import Symbol
+        >>> t = Symbol('t')
+        >>> from sympy.physics.continuum_mechanics.arch import Arch
+        >>> a = Arch((0,0),(40,0),crown_x=20,crown_y=12)
+        >>> a.apply_load(-1,'C',8,150,angle=270)
+        >>> a.apply_load(0,'D',start=20,end=40,mag=-4)
+        >>> a.apply_load(-1,'E',10,t,angle=300)
+        >>> p = a.draw()
```
Comment (AdvaitPote) [inline sympy/physics/continuum_mechanics/arch.py:664]: Arch_2

- The arch is sometimes skewed towards the bottom maybe try getting the centering better. 
- Add units for the forces and maybe bold the magnitudes
- The arrows for the distributed loads seem too close
- How about filling the arch body and member like we do in trusses

Comment (shishir-11) [reply sympy/physics/continuum_mechanics/arch.py:664]: The arch body and skew appear as such cause I am plotting two equation `self._shape_eqn` and `self._shape_eqn-0.05*lim` which gives us two lines, there is not option to give a line width . I will be fixing the arrow closeness. I would like to know if there is some way to plot curves to have fill or width.

Comment (shishir-11) [reply sympy/physics/continuum_mechanics/arch.py:664]: image
The arrows are now at a more suitable distance.

**Comment by AdvaitPote** [comment]
Works for me. Just adding the units is remaining I believe. Also try bolding the text like the forces it doesn't seem visible enough.

**Comment by shishir-11** [comment]
I have added the units and tried to bold the magnitude text . 
image

**Comment by shishir-11** [comment]
Is there any more work needed ? I was hoping to start with the ILD plot.

**Comment by AdvaitPote** [comment]
I had my convocation this weekend and so was travelling. Changes are done from my side

@Ishanned ?

**Comment by Ishanned** [comment]
> I had my convocation this weekend and so was travelling. Changes are done from my side
> 
> @Ishanned ?

LGTM, merging the pr 
Great work @shishir-11 !

**Pull request merged**